functional/FunctionalTest.java

```diff
@@ -2193,31 +2194,30 @@ public static void presignedGetObject_test1() throws Exception {
       }
 
       String urlString =
-          client.presignedGetObject(
-              PresignedGetObjectArgs.builder().bucket(bucketName).object(objectName).build());
+          client.getPresignedObjectUrl(
+              GetPresignedObjectUrlArgs.builder()
+                  .method(Method.GET)
+                  .bucket(bucketName)
+                  .object(objectName)
+                  .build());
 
       byte[] outBytes = readObject(urlString);
       if (!Arrays.equals(inBytes, outBytes)) {
         throw new Exception("object content differs");
       }
 
       client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(objectName).build());
-      mintSuccessLog("presignedGetObject(String bucketName, String objectName)", null, startTime);
+      mintSuccessLog(methodName, "presigned get object", startTime);
```

Comment:
The second argument is supposed to be the arguments passed to the method. So either pass `null` or actual values from the args.